### ISSUE TITLE
Problem inserting logo to help file header

### ISSUE THREAD

**Issue by marvelpo** [EWSoftware/SHFB#402]
Hello,

I am heaving troubles inserting logo to header. I followed instructions from Transform Args>logoFile parameter but the picture is still missing (displayed only alt text) when the project is built. I am placing the source file to the project/help/icons but the builder keeps deleting it.

**Comment by EWSoftware** [comment]
See the FAQ: [How do I add a logo to the header in my help file topics?](http://ewsoftware.github.io/SHFB/html/f5b58289-dd25-45b6-b616-5b8af6ba3739.htm#AddLogo)

You probably need to add the *icons\* folder to your SHFB project and then add the logo file to the folder in the project.
